services/gateway-api/src/circuit_breaker/test_circuit_breaker.py

```
@@ -0,0 +1,538 @@
+
+@pytest.mark.asyncio
+async def test_circuit_breaker_context_manager():
+    """Test circuit breaker context manager syntax"""
+    breaker = CircuitBreaker(
+        name="test_context",
+        config=CircuitBreakerConfig(failure_threshold=3)
+    )
+
+    async with breaker:
+        # Do some work
+        result = "context_result"
```

Ruff (0.14.8)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)